src/actions/coordinator.spec.ts

```diff
@@ -0,0 +1,41 @@
+import assert from 'assert'
+import { ITinlake } from '../types/tinlake'
+import { createTinlake } from '../test/utils'
+import testConfig from '../test/config'
+
+let tinlake: ITinlake
+
+describe('coordinator tests', async () => {
+  before(async () => {
+    tinlake = createTinlake(testConfig.godAccount, testConfig)
+  })
+
+  describe('epoch solver', async () => {
```

Comment:
Can we add some more tests?

One time where we need to stop because of maxReserve and one other test where it is not possible to find a solution because the initial conditions are not healthy?